### DISCUSSION TITLE
Skin Followpoints not disappearing at the same rate as stable.

### DISCUSSION THREAD

**Discussion by TheFantasticLoki** [ppy/osu#17072] [Q&A] [answered]
Video file was too big cause of the quality I record at so here is the Streamable Link to the video showcasing the problem: https://streamable.com/bru6as EDIT: Updated video with my mic properly recorded.
As you can see in the first clip on the replay the followpoints fade out is almost always with my cursor but on the second part in lazer autoplay on the first jump section the first followpoint doesn't fully fade out till after the last (4th hitcircle) note has been hit in the 4 note jump pattern. Makes it harder to read especially on high note density maps with large spacing.
Not sure exactly what is causing this but I feel it has to do with the fade out animation maybe not being based off the skin.ini framerate or something to that effect.

**Comment by TheFantasticLoki** [comment]
Making a comment since my mic didn't record to the correct track in the video (my bad). 
The first replay in the video is on stable and you can clearly see that the follow point disappear fairly fast, usually as my cursor is going past it depending on jump spacing. But in the second play I am switched to lazer (latest release) using the same skin on the same map. As you can clearly see the follow points fade out much later than they do on stable and this makes it confusing to read a lot of maps especially if you are already very used to the follow points on stable.

**Comment by bdach** [comment]
please link the skin

**Reply by TheFantasticLoki** [reply]
Alright here is the link to the latest version of my skin that was used in the clip @bdach : https://drive.google.com/file/d/1U4q-nMiq2B2CZQ0blN6gmcotziduzIa7/view?usp=sharing

And as a just in case measure here is the link to my skins folder with all previous versions of the skin tho not much has changed followpoint wise: https://drive.google.com/drive/folders/1Kz2ag71kdRZ6Guy5WSxzQHl_2C9p5uHm?usp=sharing

Lmk if there is anything else you need like maybe game setting wise idk.

Edit: Has anything been found about this issue?

**Reply by TheFantasticLoki** [reply]
Also skin is HD, animated, and supports all official gamemodes and yoso ruleset so sorry for large size.

**Reply by TheFantasticLoki** [reply]
@bdach Is this issue something specific to my skin or something in lazer itself? If it's something with my skin can you tell me what specifically I need to change? Everytime I switch from stable to lazer this issue really errks me similar to how peppy feels about input on key up. I just wanted to mention you again as this didn't even get a label and was curious if it was something specific to my skin.

**Reply by bdach** [reply]
i haven't checked yet sorry. this is on my notification list to check at some point. i will get around to it eventually.

**Reply by TheFantasticLoki** [reply]
Bumping this since it's been 12 days without activity. Any new developments?

**Reply by frenzibyte** [reply]
Please never do that. As *explicitly* mentioned above, it is tracked in notifications and just needs free time to be looked at, there's no reason for "bumping".

**Comment by peppy** [comment, marked as answer]
Ignoring everything in this skin (oh my god it hurts, the eyes, the brain, the disk while trying to import), the issue here is that `followpoint-49.png` and `50` don't exist.

The only reason your skin works as it does is because follow point animations apparently looped on stable while they don't on lazer. This will be resolved via https://github.com/ppy/osu/pull/17387.

**Reply by TheFantasticLoki** [reply]
Thank you for pointing this out, I checked and apparently I've been missing those 2 files since V1 of my skin but since it hadn't caused issues on stable I never noticed. I replaced the 2 files and it did fix my issue so I thank you for taking the time to post the information, even with the rather hurtful comments tho I can agree on the import issue, since it is rather large.
